moveit_core/utils/include/moveit/utils/moveit_error_code.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <moveit_msgs/MoveItErrorCodes.h>
+
+namespace moveit
+{
+namespace core
+{
+/**
+ * @brief a wrapper around moveit_msgs::MoveItErrorCodes to make it easier to return an error code message from a function
+ */
+class MoveItErrorCode : public moveit_msgs::MoveItErrorCodes
+{
+public:
+  MoveItErrorCode()
+  {
+    val = 0;
+  }
+  MoveItErrorCode(int code)
```

Comment:
Shouldn't this constructor have an `explicit`.

Reply:
I don't think so. I would like `moveit_msgs::MoveItErrorCode::SUCCESS` to convert implicitly to the class when the class is expected.

Reply:
the important part is preserving #670 (having the bool operator explicit) to avoid downcasting to bool.